ui/src/notebooks/pipes/Data/FilterTags.tsx

```diff
@@ -64,6 +67,7 @@ const constructFilters = (value: string, type: string) => {
 const FilterTags: FC = () => {
   const {data, update} = useContext(PipeContext)
   const handleDeleteFilter = (type: string, name: string) => {
+    event(`Removing ${type} By Label in Flow Query Builder`)
```

Comment:
for this one, make the name more general and throw that type into the tags attached to the event